src/main/java/backend/resource/TurboLabel.java

```diff
@@ -211,4 +223,130 @@ public int hashCode() {
     public int compareTo(TurboLabel o) {
         return actualName.compareTo(o.getActualName());
     }
+
+    /**
+     * Returns the TurboLabel in labels that matches labelName
+     * Assumption: the labelName matches exactly 1 TurboLabel
+     *
+     * @param labels
+     * @param labelName
+     * @return
+     */
+    public static TurboLabel getMatchingTurboLabel(List<TurboLabel> labels, String labelName) {
+        assert labels.stream()
+                .filter(label -> label.getActualName().equals(labelName))
+                .findFirst()
+                .isPresent();
+        return labels.stream()
+                .filter(label -> label.getActualName().equals(labelName))
+                .findFirst()
+                .get();
+    }
+
+    public static List<String> getLabelsNameList(List<TurboLabel> labels) {
+        return labels.stream()
+                .map(TurboLabel::getActualName)
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Returns the list of labels which name contains labelName
+     * @param repoLabels
+     * @param labelName
+     * @return
+     */
+    public static List<String> filterByPartialName(List<String> repoLabels, String labelName) {
```

Comment:
Wouldn't it be better to name it queryLabelName since it does not need to be partial ?